core/api/service/api_service.cpp

```diff
@@ -33,7 +33,7 @@ namespace kagome::api {
       std::shared_ptr<api::RpcThreadPool> thread_pool,
       std::vector<std::shared_ptr<Listener>> listeners,
       std::shared_ptr<JRpcServer> server,
-      gsl::span<std::shared_ptr<JRpcProcessor>> processors,
+      const std::vector<std::shared_ptr<JRpcProcessor>> &processors,
```

Comment:
Why?

Reply:
The same effect without using third-party components.